appengine/standard/firebase/firetactoe/README.md

```diff
@@ -15,7 +15,7 @@ installed. You'll need this to test and deploy your App Engine app.
   console](https://firebase.google.com/console)
 * In the Overview section, click 'Add Firebase to your web app' and replace the
   contents of the file
-  [`templates/_firebase_config.html`](templates/_firebase_config.html) with the
+  [`templates/_firebase_config.json`](templates/_firebase_config.json) with the
```

Comment:
I took one look at the regex and my head nearly exploded :). Would prefer json, but just noticed that you inject the snippet into the html, which is simpler than having to copy values twice. And I broke the sample when I removed the snippet file for the pull request. Let's go with what you had. Perhaps you could just add a really good comment about what you're doing. The regex really does make it look unnecessarily complex.